lib/optimize/DedupePlugin.js

```diff
@@ -207,6 +212,11 @@ DedupePlugin.prototype.apply = function(compiler) {
 							"}(modules[i]));",
 							"break;"
 						]),
+						"case \"undefined\":",
+						this.indent([
+							"// Module is skipped in this bundle",
+							"break;"
+						]),
 						"default:",
```

Comment:
Could you rewrite the `default` branch to `case "number":`? This should have the same effect, but with smaller size. Then we could also remove the `"function"` branch. everything should be ignored by default.